sunpy/coordinates/ephemeris.py

```diff
@@ -22,7 +32,7 @@
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def get_body_heliographic_stonyhurst(body, time='now', observer=None):
+def get_body_heliographic_stonyhurst(body, time='now', observer=None, include_velocity=False):
```

Comment:
The `include_velocity` kwarg here feels like a good candidate for a keyword only argument to me. Specifying a boolean as a positional argument is really meaningless when reading the code?

Reply:
Done.  I've moved the docstring documentation for the keyword out of "Parameters" to a new section "Keyword Arguments" to highlight that it is a keyword-only argument.